File: GSI QA testing tool/UI/UI_030_WorkersCompensation.cs
using OpenQA.Selenium;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GSI_QA_testing_tool.UI
{
    internal class UI_030_WorkersCompensation
    {
        public static void GoTo(IWebDriver driver)
        {
            Finder.ClickIt(driver, "//label[@for='ctl00_Main_content_Wizard1_ucWorkersCompensation_rblWCReceived_1']");
            Finder.ClickIt(driver, "//input[@id='ctl00_Main_content_Wizard1_StepNavigationTemplateContainerID_StepNextButton']");

        }
    }
}

File: GSI QA testing tool/UI/UI_030_WorkersCompensation.vb
Imports OpenQA.Selenium
Imports System
Imports System.Collections.Generic
Imports System.Diagnostics
Imports System.Linq
Imports System.Text
Imports System.Threading
Imports System.Threading.Tasks

Namespace GSI_QA_testing_tool.UI
    Friend Class UI_030_WorkersCompensation
        Public Shared Sub [GoTo](ByVal driver As IWebDriver, ByVal js As IJavaScriptExecutor)
            Thread.Sleep(300)
            If Data._StopAt.Contains("Worker's Compensation") Then
                Data.StopAtDialog()
            End If

            If Finder.FindItByCSS(driver, "*[for='ctl00_Main_content_Wizard1_ucWorkersCompensation_rblWCReceived_1']") Then
                Debug.WriteLine("WorkersCompensation is On")
            Else
                Debug.WriteLine("WorkersCompensation is Off")
                Return
            End If

            Finder.JSClickIt(driver, "#ctl00_Main_content_Wizard1_ucWorkersCompensation_rblWCReceived_1", js)
            js.ExecuteScript("arguments[0].click();", driver.FindElement(By.XPath("//input[@id='ctl00_Main_content_Wizard1_StepNavigationTemplateContainerID_StepNextButton']")))
        End Sub
    End Class
End Namespace
